src/styles/blocks/calendar.scss

```diff
@@ -0,0 +1,51 @@
+body {
+  margin: 0;
+}
+
+.calendar {
+  display: flex;
+  flex-wrap: wrap;
+  width: calc(7 * $day-size + 6 * $gap + 2 * $padding);
+  padding: 5px;
+  padding: $padding;
```

Comment:
The `padding` property is set twice on the `.calendar` class. The second declaration using `$padding` will override the first one. Ensure that `$padding` is defined and intended to be used here.